@types/helpers.d.ts

```diff
@@ -25,4 +25,5 @@ export interface UserSession extends Session {
 
 export interface Context {
   req: LoggedRequest
+  res: Response
```

Comment:
We should be using Next's API types here instead of Express. `NextApiRequest` and `NextApiResponse`.

https://nextjs.org/docs/basic-features/typescript#api-routes
https://nextjs.org/docs/api-routes/api-middlewares#extending-the-reqres-objects-with-typescript